gym_ai2thor/tasks.py

```diff
@@ -36,7 +41,7 @@ def __init__(self, config):
         self.task_config = config
         self.max_episode_length = config['max_episode_length'] \
             if 'max_episode_length' in config else 1000
-        self.movement_reward = config['movement_reward'] if 'movement_reward' in config else 0
+        self.movement_reward = config['movement_reward'] if 'movement_reward' in config else -1
```

Comment:
-1 is a common practice and helps learn faster. If you prefer I can roll back to 0 and add the movement reward to rainbow_config so that it is clear how I trained.

Reply:
how about -0.1 or -0.01?
-1 is really really big. 
but sure you can add it to the config